Tests/test_file_png.py

```diff
@@ -620,16 +613,536 @@ def test_exif_argument(self):
         with Image.open(test_file) as reloaded:
             self.assertEqual(reloaded.info["exif"], b"Exif\x00\x00exifstring")
 
-    @unittest.skipUnless(
-        HAVE_WEBP and _webp.HAVE_WEBPANIM, "WebP support not installed with animation"
-    )
-    def test_apng(self):
-        with Image.open("Tests/images/iss634.apng") as im:
+    # APNG browser support tests and fixtures via:
```

Comment:
These tests double the size of this file, let's move them to their own `test_file_apng.py` or `test_file_png_apng.py`.

Reply:
The requested documentation changes are fixed in df42b63